src/meteodatalab/operators/omega_slope.py

```diff
@@ -30,7 +31,15 @@ def cumdiff(A, axis):
 def omega_slope(
     ps: xr.DataArray, etadot: xr.DataArray, ak: xr.DataArray, bk: xr.DataArray
 ):
-    """Compute omega slope."""
+    """Compute the omega slope.
+
+    Converts ECMWF etadot (deta/dt) to etadot * dp/deta, required by FLEXPART.
+
+    Note:
```

Comment:
Should be plural and without column
https://numpydoc.readthedocs.io/en/latest/format.html#notes